iocage/lib/LaunchableResource.py

```diff
@@ -69,17 +72,6 @@ def dataset_name(self, value: str) -> None:
         )
 
     @property
-    def dataset(self) -> libzfs.ZFSDataset:
-        if self._dataset is None:
-            self._dataset = self.zfs.get_dataset(self.dataset_name)
-
-        return self._dataset
-
-    @dataset.setter
-    def dataset(self, value: libzfs.ZFSDataset):
-        self._set_dataset(value)
-
-    @property
```

Comment:
what happened here?

Reply:
It's implemented by the parent class. This overrides are no longer needed.

Reply:
aye